lib/licensee/content_helper.rb

```diff
@@ -24,6 +24,9 @@ module ContentHelper
       developed_by:        /#{START_REGEX}developed by:.*?\n\n/im,
       quote_begin:         /[`'"‘“]/,
       quote_end:           /[`'"’”]/,
+      cc_legal_code:       /^\s*Creative Commons Legal Code\s*$/i,
+      cc0_info:            /For more information, please see\s*\S+zero\S+/im,
+      cc0_disclaimer:      /CREATIVE COMMONS CORPORATION.*?\n\n/im,
```

Comment:
Any concern that this might be over inclusive? Anything after any text that reads `CREATIVE COMMONS CORPORATION` would be stripped, if I'm reading correctly? Wondering if we can't match the exact normalized text?

Reply:
Anything up to a new paragraph. I'm not particularly concerned about it being over inclusive. The whole optional paragraph is long, but if you'd prefer I could certainly change to look for a match agains the whole (normalized) paragraph:

> CREATIVE COMMONS CORPORATION IS NOT A LAW FIRM AND DOES NOT PROVIDE LEGAL SERVICES. DISTRIBUTION OF THIS DOCUMENT DOES NOT CREATE AN ATTORNEY-CLIENT RELATIONSHIP. CREATIVE COMMONS PROVIDES THIS INFORMATION ON AN "AS-IS" BASIS. CREATIVE COMMONS MAKES NO WARRANTIES REGARDING THE USE OF THIS DOCUMENT OR THE INFORMATION OR WORKS PROVIDED HEREUNDER, AND DISCLAIMS LIABILITY FOR DAMAGES RESULTING FROM THE USE OF THIS DOCUMENT OR THE INFORMATION OR WORKS PROVIDED HEREUNDER.